packages/data-visualization/src/StatProgress/StatProgress.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import { Flex, Box, Text } from '@kodiak-ui/primitives'
+import { Progress, ProgressThumb } from '@kodiak-ui/progress'
+import { SxStyleProp } from '@kodiak-ui/core'
+
+type StatProgressProps = {
+  label?: React.ReactNode
+  labelRight?: React.ReactNode
+  showProgressThumb?: boolean
+  color?: string
+  progressSx?: SxStyleProp
+} & React.ComponentProps<typeof Progress>
+
+export function StatProgress({
+  color,
+  value,
+  min,
+  max,
+  label,
+  labelRight,
+  showProgressThumb,
+  progressSx,
+}: StatProgressProps) {
+  return (
+    <Box variantKey="statProgress">
+      <Flex
+        __base={{
+          alignItems: 'center',
+          mb: 2,
+          justifyContent: 'space-between',
+        }}
+      >
+        <Text as="span" sx={{ fontWeight: 'semiBold' }}>
```

Comment:
:thinking: maybe `{label && ` to avoid creating an empty DOM element

Reply:
Let me try some things but it'll need some layout tweaks if someone provides only the `labelRight` since the `space-between` is what's getting the right item to position where it is.

Reply:
also it'll have to check more of the `undefined` states because it's possible that the user provides `0` and we'd probably want to show that.

Reply:
Think we'll leave it since the empty dom element is a bit easier than trying to update the layout.  I will change it to a box though because a `span` complains when there's different nesting elements inside of it.